src/control/cmd/agent/main.go

```diff
@@ -46,6 +46,7 @@ var daosVersion string
 const (
 	agentSockName        = "agent.sock"
 	daosAgentDrpcSockEnv = "DAOS_AGENT_DRPC_DIR"
+	defaultConfigPath    = "../etc/daos_agent.yml"
```

Comment:
A little concerned about the default path being relative, but I guess this covers the normal installation cases, both in tree and out.

Reply:
Unfortunately that is the way all of the checks are. Its not my ideal either but it should work for now.  This is just a hack so we support running from the installed tree during a build. It would be nice if we could conditionally change these paths during an RPM build to use proper system paths. That however is out of scope for this PR.

Reply:
> It would be nice if we could conditionally change these paths during an RPM build to use proper system paths.

I'm working on that as part of a fix for DAOS-4134. My plan is to set CONF_DIR to `$PREFIX/etc` for source builds and `%{_sysconfdir}` for RPM builds. Finding scons a bit fiddly but getting there...